Documentation/docs/README.md

```diff
@@ -1,18 +1,51 @@
-# Sphinx Documentation
+## How to contribute documentation improvements
 
-This is the source of the [ITK docs](docs.itk.org) website.
+Thank you for improving our documentation! Every addition made is read
+thousands of times and has a tremendous impact.
 
-## Build environment
+These are the sources for the [ITK docs](docs.itk.org)
+[sphinx](https://www.sphinx-doc.org/) website.
+
+The contribution process generally follows our [contributing
+guidelines](../../CONTRIBUTING.md).
+
+To preview documentation changes, three options are available.
+
+### Option 0: Edit on GitHub, use the pull request preview.
```

Comment:
Nitpick: one-indexing (Option 1, 2, 3) is more standard, but doesn't need to hold up these changes